x-pack/plugins/ux/public/components/app/rum_dashboard/charts/visitor_breakdown_chart.tsx

```diff
@@ -207,6 +227,15 @@ export function getVisitorBreakdownLensAttributes({
                       },
                     ]
                   : []),
+                {
+                  range: {
+                    [localDataView.timeFieldName ?? '@timestamp']: {
+                      format: 'strict_date_optional_time',
+                      gte: start,
+                      lte: end,
+                    },
+                  },
+                },
```

Comment:
This should not be necessary, but built-in `timeRange` seems to be ignored after the change